deploy/backplane/20-cee.SubjectPermission.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-cee
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - allowFirst: true
+    clusterRoleName: backplane-cee
+    namespacesAllowedRegex: .*
```

Comment:
The backplane cee ClusterRole is bundling cluster and project scope resources.  There isn't a reason to create a SubjectPermission, it can just be a ClusterRoleBinding.

Reply:
Yes, agreed thanks!. Deleted subjectPermission and added a clusterRolebinding for CEE.